src/mgm/tdes.rs

```diff
@@ -0,0 +1,70 @@
+use zeroize::Zeroizing;
+
+/// Size of a DES key
+pub(super) const DES_LEN_DES: usize = 8;
+
+/// Size of a 3DES key
+pub(crate) const DES_LEN_3DES: usize = DES_LEN_DES * 3;
+
+/// Weak and semi weak DES keys as taken from:
```

Comment:
@baloo will we be able to replace these checks using the weak key feature in next (pre)release of the `des` crate?

Reply:
can do that.

Reply:
https://github.com/iqlusioninc/yubikey.rs/pull/601